Expose the upgrade tool in docs.daml.com (#790)
digital-asset/docs.daml.com: docs/2.8.9/docs/upgrade/automation.rst

Reviewer: I recommend breaking down the sentence "The tool is designed to accommodate: working with..." into the following list:

The tool is designed to accommodate:
- Working with a stale view of the ledger’s state, such as after a partial migration;
- Reducing effort by generating boilerplate code for unchanged templates;
- Providing idempotent operations where possible;
- Handling very large ACS sizes; and
- Robustly handling rejected ledger commands, such as by archiving a contract in the same Daml transaction that creates an upgraded replacement, ensuring that duplicate replacement contracts are not created.

Nevertheless, we have found that the pattern shown here
occurs frequently.

This section describes manual upgrading steps when deploying updated Daml

> application code, by migrating contracts from one version to another.  A

Reviewer: ```suggestion
application code, by migrating contracts from one version to another. A
```

> production grade, optimized migration tool that automates this application

Reviewer: ```suggestion
production-grade, optimized migration tool that automates this application
```

> upgrade process is available but it does require a Daml Enterprise license.

Reviewer: ```suggestion
upgrade process is available, but it requires a Daml Enterprise license.
```

Its PDF manual explains how a Daml developer implements a smart contract

> upgrade migration. The manual also  advises an operations team on how to

Reviewer: ```suggestion
upgrade migration. The manual also advises an operations team on how to
```

perform the contract migration.  

The tool is designed to accommodate: working with a stale view of the ledger’s
state (e.g., a partial migration was recently performed), reduce effort by
boilerplate code generation for unchanged templates, it has idempotent
operations where possible, very large ACS sizes, and robustly handles rejected
ledger commands (e.g., archiving a contract in the same Daml transaction that
creates an upgraded replacement helps to ensure that duplicate replacement
contracts are not created.)  Please contact your account manager or `DA

> support <support@digitalasset.com>`_ for access to the upgrade tool.

Reviewer: ```suggestion
support <support@digitalasset.com>`_ to access the upgrade tool.
```
